package.json

```diff
@@ -37,58 +70,30 @@
     "stylelint-config-standard": "^17.0.0",
     "supertest": "^3.0.0",
     "testpilot-ga": "^0.3.0",
-    "uglifyify": "^4.0.3"
+    "uglifyify": "^4.0.3",
+    "webpack": "^3.4.1"
   },
   "engines": {
     "node": ">=8.0.0"
   },
   "homepage": "https://github.com/mozilla/send/",
   "license": "MPL-2.0",
   "repository": "mozilla/send",
-  "availableLanguages": [
-    "en-US",
-    "zh-TW",
-    "zh-CN",
-    "cs",
-    "fr",
-    "fy-NL",
-    "de",
-    "hu",
-    "it",
-    "ja",
-    "kab",
-    "ms",
-    "nb-NO",
-    "nn-NO",
-    "pt-PT",
-    "pt-BR",
-    "ru",
-    "sk",
-    "sl",
-    "dsb",
-    "hsb",
-    "es-CL",
-    "es-ES",
-    "es-MX",
-    "sv-SE",
-    "tr",
-    "cy"
-  ],
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:upload": "browserify frontend/src/upload.js -g uglifyify -o public/upload.js",
-    "build:download": "browserify frontend/src/download.js -g uglifyify -o public/download.js",
-    "build:version": "node scripts/version",
     "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
+    "build:public": "npm run webpack -- # -p",
```

Comment:
This was part of the comedy of ES6 uglification... If I uncomment out that rogue `-p` flag, I get the following errata:

```
> firefox-send@1.0.2 webpack /Users/pdehaan/dev/github/mozilla/something-awesome
> webpack "-p"

Hash: a12e4c34ccf33171606b
Version: webpack 3.4.1
Time: 4236ms
      Asset    Size  Chunks                    Chunk Names
  upload.js  682 kB       0  [emitted]  [big]  upload
download.js  670 kB       1  [emitted]  [big]  download
  [45] (webpack)/buildin/global.js 509 bytes {0} {1} [built]
  [60] ./frontend/src/utils.js 2.82 kB {0} {1} [built]
 [300] ./frontend/src/common.js 934 bytes {0} {1} [built]
 [310] ./frontend/src/storage.js 2.83 kB {0} {1} [built]
 [312] multi babel-polyfill ./frontend/src/upload.js 40 bytes {0} [built]
 [313] ./frontend/src/upload.js 16.9 kB {0} [built]
 [314] ./frontend/src/fileSender.js 6.74 kB {0} [built]
 [315] multi babel-polyfill ./frontend/src/download.js 40 bytes {1} [built]
 [316] ./frontend/src/download.js 6.49 kB {1} [built]
 [317] ./frontend/src/fileReceiver.js 5.29 kB {1} [built]
    + 308 hidden modules

ERROR in upload.js from UglifyJs
Unexpected token: name (TestPilotGA) [upload.js:21531,8]

ERROR in download.js from UglifyJs
Unexpected token: name (TestPilotGA) [download.js:21531,8]

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! firefox-send@1.0.2 webpack: `webpack "-p"`
```

IIUC, **uglify-js** doesn't support ES6, and we should be using **uglify-es**. Not sure if that means we'd need to switch to something like https://github.com/noBlubb/uglify-es-webpack-plugin